app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= link_to 'New Status Update', [:new, :students, :status_update], class: 'btn btn-default'
```

Comment:
Thanks Yorick! You just might have solved a bunch of other problems with this comment.